scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -107,7 +107,22 @@ public boolean doCheckAndSend(boolean withAttachment)
         String jobStatus = jobState.getGenericInformation().get(GENERIC_INFORMATION_KEY_NOTIFICATION_EVENT);
         List<String> jobStatusList = new ArrayList<>();
         if (jobStatus != null) {
-            jobStatusList = Arrays.asList(jobStatus.toLowerCase().split("\\s*,\\s*"));
+            if ("all".equals(jobStatus.toLowerCase())) {
+                jobStatusList = Arrays.asList("JOB_CHANGE_PRIORITY",
+                                              "JOB_IN_ERROR",
+                                              "JOB_PAUSED",
+                                              "JOB_PENDING_TO_FINISHED",
+                                              "JOB_PENDING_TO_RUNNING",
+                                              "JOB_RESTARTED_FROM_ERROR",
+                                              "JOB_RESUMED",
+                                              "JOB_RUNNING_TO_FINISHED",
+                                              "JOB_SUBMITTED")
+                                      .stream()
+                                      .map(status -> status.toLowerCase())
+                                      .collect(Collectors.toList());
+            } else {
+                jobStatusList = Arrays.asList(jobStatus.toLowerCase().split("\\s*,\\s*"));
```

Comment:
And could you please describe what does this pattern mean?

Reply:
This line is what exist before. It parse the value of the key "NOTIFICATION EVENTS" in generic info. This value specifies the events that a user want to receive an email notification, and the events are separated by ",". The case of the events does not metter (you can type either lower case or upper case or even a mix)

Reply:
sorry i meant to put comment in the code

Reply:
okay